services/operatorsinfo/operatorsinfo_inmemory.go

```diff
@@ -151,21 +165,35 @@ func (ops *OperatorsInfoServiceInMemory) startServiceInGoroutine(ctx context.Con
 	}()
 }
 
-func (ops *OperatorsInfoServiceInMemory) queryPastRegisteredOperatorEventsAndFillDb(ctx context.Context) {
+func (ops *OperatorsInfoServiceInMemory) queryPastRegisteredOperatorEventsAndFillDb(ctx context.Context) error {
 	// Querying with nil startBlock and stopBlock will return all events. It doesn't matter if we query some events that we will receive again in the websocket,
 	// since we will just overwrite the pubkey dict with the same values.
-	alreadyRegisteredOperatorAddrs, alreadyRegisteredOperatorPubkeys, err := ops.avsRegistryReader.QueryExistingRegisteredOperatorPubKeys(ctx, nil, nil)
-	if err != nil {
-		ops.logger.Error("Fatal error querying existing registered operators", "err", err, "service", "OperatorPubkeysServiceInMemory")
-		panic(err)
-	}
-	ops.logger.Debug("List of queried operator registration events in blsApkRegistry", "alreadyRegisteredOperatorAddr", alreadyRegisteredOperatorAddrs, "service", "OperatorPubkeysServiceInMemory")
+	wg := sync.WaitGroup{}
+	var alreadyRegisteredOperatorAddrs []common.Address
+	var alreadyRegisteredOperatorPubkeys []types.OperatorPubkeys
+	var pubkeysErr error
+
+	// we make both Queries in parallel because they take time and we don't want to wait for one to finish before starting the other
+	wg.Add(2)
+	go func() {
+		alreadyRegisteredOperatorAddrs, alreadyRegisteredOperatorPubkeys, pubkeysErr = ops.avsRegistryReader.QueryExistingRegisteredOperatorPubKeys(ctx, nil, nil, ops.logFilterQueryBlockRange)
+		wg.Done()
+	}()
+	var socketsMap map[types.OperatorId]types.Socket
+	var socketsErr error
+	go func() {
+		socketsMap, socketsErr = ops.avsRegistryReader.QueryExistingRegisteredOperatorSockets(ctx, nil, nil, ops.logFilterQueryBlockRange)
+		wg.Done()
+	}()
 
-	socketsMap, err := ops.avsRegistryReader.QueryExistingRegisteredOperatorSockets(ctx, nil, nil)
-	if err != nil {
-		ops.logger.Error("Fatal error querying existing registered operator sockets", "err", err, "service", "OperatorPubkeysServiceInMemory")
-		panic(err)
+	wg.Wait()
+	if pubkeysErr != nil {
+		return errors.Join(errors.New("error querying existing registered operators"), pubkeysErr)
```

Comment:
we have a wrapErr function in utils now right? should use that

Reply:
[3f04092](https://github.com/Layr-Labs/eigensdk-go/pull/239/commits/3f040927bc5486d41df9e8f69ef43878c79c82b7)

Reply:
side note: actually don't like utils.wrapErr, wish it was in an errors module, so that we would do errors.wrapErr instead. we could import the sdk errors module as sdkerrors.wrapErr or something